### PR TITLE
fix(rule): 'NgModule' not being handled properly for contextual-lifecycle and contextual-decorators

### PR DESCRIPTION
**Pull request by wKoza**

### CODE REVIEW

File: test/contextualLifecycleRule.spec.ts [hunk 1]
```diff
@@ -150,6 +150,170 @@ describe(ruleName, () => {
       });
     });
 
+    describe('failure', () => {
```
Comment (rafaelss95) [inline test/contextualLifecycleRule.spec.ts]:
```suggestion

```

We already have a _failure_ **describe** block. You can put the `NgModule` block into it.

Comment (rafaelss95) [inline test/contextualLifecycleRule.spec.ts]:
```suggestion

```

We already have a _failure_ **describe** block. You can put the `NgModule` block into it.

**Comment by rafaelss95** [comment]
Nice catch! ✅ 

I guess this change also fix the `contextual-decorator`, right? If so, let's rephrase the commit.

**Comment by wKoza** [comment]
Actually, we kill two birds with one stone.

**Comment by rafaelss95** [comment]
LGMT!

**Pull request merged**